agent/app/agent_capability.go

```diff
@@ -543,11 +543,14 @@ func (agent *ecsAgent) appendEBSTaskAttachCapabilities(capabilities []*ecs.Attri
 	return capabilities
 }
 
+// TODO Remove linter directive below when the function becomes used
```

Comment:
If we are not already, can we also make sure to track this TODO as a task internally (separate from this code comment)? Just want to ensure this removal does not get missed nor left behind in the "sea of lingering TODOs" currently in the code base. Thanks!

Reply:
Yep we are tracking internally.

Reply:
I will be submitting a PR to a feature branch with that change once this is merged.